scripts/sysman/common.yml

```diff
@@ -211,6 +211,9 @@ etors:
     - name: DEVICE_UUID
       desc: $s_uuid_t
       version: "1.7"
+    - name: MEM_EXT_BANDWIDTH
+      desc: $s_mem_ext_bandwidth_t
+      version: "1.7"
```

Comment:
@wdamon-intel @jandres742 have retained version as 1.7 here since the extension was already introduced in 1l7 and intention here is to fix the stype. please let me know if this is fine or version needs to be updated